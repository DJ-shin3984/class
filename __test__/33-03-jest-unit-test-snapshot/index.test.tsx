import JestUnitTestSnapshotPage from "../../pages/section33/33-03-jest-unit-test-snapshot"
import { render } from "@testing-library/react"
import "@testing-library/jest-dom"

it("기존 사진이랑 변경점 있는지 확인-스냅샷 테스트",() => {
  const result = render(<JestUnitTestSnapshotPage/>)
  expect(result.container).toMatchSnapshot()
})