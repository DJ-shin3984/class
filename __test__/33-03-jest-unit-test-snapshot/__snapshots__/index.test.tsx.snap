// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`기존 사진이랑 변경점 있는지 확인-스냅샷 테스트 1`] = `
<div>
  <div>
    철수는 13 입니다.
  </div>
  철수의 취미 입력
  <input
    type="text"
  />
  <button>
    철수랑 놀기
  </button>
</div>
`;
